examples/aks-spring-music/porter.yaml

```diff
@@ -35,6 +36,15 @@ install:
         - "--tenant" 
         - "{{ bundle.credentials.TENANT_ID}}"
 
+  - exec:
```

Comment:
@jeremyrickard Now that we have the az mixin, long term (not this PR) do we want to update this to use the az mixin?